stubs/networkx/networkx/drawing/nx_pylab.pyi

```diff
@@ -25,7 +25,7 @@ def draw_networkx_nodes(
     pos,
     nodelist: Collection[Incomplete] | None = None,
     node_size: Incomplete | int = 300,
-    node_color: str = "#1f78b4",
+    node_color: str | list[str] = "#1f78b4",
```

Comment:
I don't have any experience with networkx, but looking at the documentation and code, it's probably better to use `Sequence` (must be imported from `collections.abc`) instead, as that allows a covariant use. We might even have a protocol in `_typeshed` that matches even better, but that would need more research.

```suggestion
    node_color: str | Sequence[str] = "#1f78b4",
```

Reply:
Thanks!